src/block_cache/block_mgr.c

```diff
@@ -205,10 +207,11 @@ __bm_checkpoint_last(WT_BM *bm, WT_SESSION_IMPL *session, char **metadatap, char
  *     Write a buffer into a block, creating a checkpoint; readonly version.
  */
 static int
-__bm_checkpoint_readonly(
-  WT_BM *bm, WT_SESSION_IMPL *session, WT_ITEM *buf, WT_CKPT *ckptbase, bool data_checksum)
+__bm_checkpoint_readonly(WT_BM *bm, WT_SESSION_IMPL *session, WT_ITEM *buf,
+  WT_PAGE_BLOCK_META *block_meta, WT_CKPT *ckptbase, bool data_checksum)
 {
     WT_UNUSED(buf);
+    WT_UNUSED(block_meta);
```

Comment:
For future readers, see why we are doing this [here](https://github.com/wiredtiger/wiredtiger/pull/11960/files#r2108086871)